opentelekomcloud/resource_opentelekomcloud_compute_instance_v2.go

```diff
@@ -428,11 +432,20 @@ func resourceComputeInstanceV2Create(d *schema.ResourceData, meta interface{}) e
 
 	configDrive := d.Get("config_drive").(bool)
 
+	rawSecGroups := d.Get("security_groups").(*schema.Set).List()
+	sgIDs, err := resourceInstanceSecGroupsV2(rawSecGroups, computeClient)
+	if err := d.Set("security_groups", sgIDs); err != nil {
+		return fmt.Errorf("failed to set IDs into security_groups")
+	}
+	if err != nil {
```

Comment:
may you up this `if err!=nil`? after `sgIDs, err := resourceInstanceSecGroupsV2(rawSecGroups, computeClient)`